doc/examples/fiber_to_bundle_coherence.py

```diff
@@ -142,8 +142,9 @@
 
 streamlines = LocalTracking(prob_dg, classifier, seeds, affine, step_size=.5)
 
-# Compute streamlines and store as a list.
-streamlines = list(streamlines)
+# Compute streamlines.
+from dipy.tracking.streamline import Streamlines
```

Comment:
Maybe import from nibabel instead?

Reply:
For a tutorial, I do not find really user friendly the syntax `from nibabel.streamlines import ArraySequence as Streamlines`

However, I do prefer the name `Streamlines` over `ArraySequence`. So I just imported from DIPY.